src/main/java/com/reckue/post/models/Rating.java

```diff
@@ -0,0 +1,23 @@
+package com.reckue.post.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Document
+public class Rating {
+
+   @Id
+    private String id;
+
+    private String userId;
```

Comment:
Please, add javadoc before class.
May be it is better to set variables User userId and Post postId? But i don't insist.